lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -213,6 +213,9 @@ sub validate_syntax {
         }
 
         foreach my $ns_ip ( @{ $syntax_input->{nameservers} } ) {
+            return { status => 'nok', message => encode_entities( "Invalid IP address format" ) }
```

Comment:
I think it would be reasonable to have the same message here as below:

message => encode_entities( "Invalid IP address: [$ns_ip->{ip}]" )